src/pages/Medicine.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useEffect } from "react";
+import { BackButton, FormMedicine } from "../components ";
+import { Link, NavLink, Router, useNavigate } from "react-router-dom";
+import { useLocalStorage } from "../hooks/useLocalStorage";
+
+const Medicine = () => {
+  const [medicineName, setMedicineName] = useState("");
+  const [medicineQtyOne, setMedicineQtyOne] = useState("");
+  const [medicineDate, setMedicineDate] = useState("");
+  const [medicineTime, setMedicineTime] = useState("");
+  const [medicineQtyTotal, setMedicineQtyTotal] = useState("");
+  const [medicineDose, setMedicineDose] = useState("");
+  const [medicineInterval, setMedicineInterval] = useState("");
+  const [medicineStartDate, setMedicineStartDate] = useState("");
+  const [medicineStartTime, setMedicineStartTime] = useState("");
```

Comment:
We could consider having a full on Medicine state object here. With this much state, it will get hard to manage. Otherwise you could also try using a library like Formik or React hook form